<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>400</width><height>250</height></rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Device</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QFormLayout">
     <item row="0" column="0"><widget class="QLabel"><property name="text"><string>COM Port:</string></property></widget></item>
     <item row="0" column="1"><widget class="QComboBox" name="comPortCombo"/></item>
     <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Baud Rate:</string></property></widget></item>
     <item row="1" column="1"><widget class="QComboBox" name="baudCombo"/></item>
     <item row="2" column="0"><widget class="QLabel"><property name="text"><string>Data Bits:</string></property></widget></item>
     <item row="2" column="1"><widget class="QComboBox" name="dataBitsCombo"/></item>
     <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Parity:</string></property></widget></item>
     <item row="3" column="1"><widget class="QComboBox" name="parityCombo"/></item>
     <item row="4" column="0"><widget class="QLabel"><property name="text"><string>Stop Bits:</string></property></widget></item>
     <item row="4" column="1"><widget class="QComboBox" name="stopBitsCombo"/></item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="connectBtn">
     <property name="text"><string>Connect</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
